docs/src/pages/components/authenticator/index.page.mdx

```diff
@@ -371,6 +373,29 @@ The following example customizes these slots with:
   />
 </Example>
 
+### Override Function Calls
+
+You can override the call to `signUp`, `signIn`, `confirmSignIn`, `confirmSignUp`, `forgotPassword` and `forgotPasswordSubmit` functions.
+To override a call you must create a new `services` object with an `async` `handle*` function that returns an `aws-amplify` `Auth.*` promise.
+
+The service object must then be passed into the `authenticator` component as a `services` prop. For example, let's imagine you'd like to lowercase the `username` and the `email` attributes during `signUp`.
+This would be overriden like so:
+
+<Fragment>
+  {({ platform }) => import(`./overrides/username.${platform}.mdx`)}
```

Comment:
Question on this example. It's not obvious to me from just reading the example where the `Auth.signUp` function is coming from. Should we include the `import {Auth} from 'aws-amplify'` in this example?

Reply:
Yeah, that's a good idea, I'll add that.

Reply:
Ok that's back in, I'm just showing the whole file, like we do in Angular and Vue.